# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/squeak/hotfix-remove-npsqueak.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- ../CMakeLists.txt.vanilla	2025-09-10 16:47:11.633151442 +0200
+++ ../CMakeLists.txt	2025-09-10 16:47:18.718113021 +0200
@@ -155,8 +155,6 @@
 
 BUILD_PLUGINS ()
 
-ADD_SUBDIRECTORY (${unix}/npsqueak)
-
 SET (cmake ${unix}/../../cmake)
 
 IF (EXISTS ${cmake}/Packages.cmake)
